src/components/OfferCard/components/InclusionsGroup/InclusionsGroup.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, FeatureListItem, NakedButton, Text } from 'components';
+
+const InclusionGroup = ({
```

Comment:
roo-ui being a component library should remain generic without any business/product knowledge, so thinking this should be renamed to align with `FeatureListItem`?

`FeatureListItem` is mainly used for inclusions but it was introduced in roo-ui with a generic name

```suggestion
const FeatureListGroup = ({
```

Same for `OfferCard` - rename to a generic name?